src/Classes/Daemons/MyRadio_TrackAndTraceDaemon.php

```diff
@@ -71,6 +71,13 @@ private static function generateTrackAndTraceReport()
             }
         }
 
+        usort($data, function ($a, $b) {
+            if ($a["unix"] == $b["unix"]) {
+                return 0;
+            }
+            return ($a["unix"] < $b["unix"]) ? -1 : 1;
```

Comment:
Can (in theory) just be
```suggestion
            return $a["unix"] - $b["unix"];
```

(the proof is left as an exercise to the reader :P )

Reply:
imma blame you if it don't work :)